client/src/includes/a11y-result.test.ts

```diff
@@ -92,8 +92,9 @@ describe('addCustomChecks', () => {
 });
 
 // Options for checkImageAltText function
+// Permitted values are read from `get_allowed_image_extensions()`
 const options = {
-  pattern: '\\.(avif|gif|jpg|jpeg|png|svg|webp)$|_',
```

Comment:
Removed `svg` because it [isn't returned by `get_allowed_image_extensions()` by default](https://github.com/wagtail/wagtail/blob/fd9b1ca22b5ae1167d54a9c980f48c27792d4403/wagtail/images/fields.py#L18)